src/ilasm/grammar_after.cpp

```diff
@@ -1500,6 +1501,27 @@ AsmParse::~AsmParse()
     while(m_ANSLast.POP());
 }
 
+#ifdef FEATURE_PAL
+// This is a simplified implementation of IsTextUnicode.
+BOOL IsTextUnicode(CONST VOID* lpv, int iSize, LPINT lpiResult)
+{
+    *lpiResult = 0;
+
+    if (iSize < 2) return FALSE;
+
+    BYTE * p = (BYTE *)lpv;
+
+    // Check for Unicode BOM
```

Comment:
nit: this is formally called Utf16LE (little endian). Unicode is a general name for character set which codepoints are being encoded